for some reason, upstream found a need to force a C linker, but the linked-in 
libraries are c++ -- so the link fails when clang is called

we remove this force to a C linker, and cmake picks the right linker, clang++,
and the link succeeds

kencu@macports.org


diff --git external/squirrel/sq/CMakeLists.txt external/squirrel/sq/CMakeLists.txt
index daadb2d..a2b49ae 100644
--- external/squirrel/sq/CMakeLists.txt
+++ external/squirrel/sq/CMakeLists.txt
@@ -12,7 +12,6 @@ endif()
 
 if(NOT DEFINED DISABLE_STATIC)
   add_executable(sq_static sq.c)
-  set_target_properties(sq_static PROPERTIES LINKER_LANGUAGE C)
   target_link_libraries(sq_static squirrel_static sqstdlib_static)
   if(NOT DEFINED SQ_DISABLE_INSTALLER)
     install(TARGETS sq_static RUNTIME DESTINATION ${INSTALL_BIN_DIR})
